Doc/library/shutil.rst

```diff
@@ -531,12 +531,13 @@ provided.  They rely on the :mod:`zipfile` and :mod:`tarfile` modules.
    available), or "xztar" (if the :mod:`lzma` module is available).
 
    *root_dir* is a directory that will be the root directory of the
-   archive; for example, we typically chdir into *root_dir* before creating the
-   archive.
+   archive, all paths in the archive will be relative to it; for example,
+   we typically chdir into *root_dir* before creating the archive.
 
    *base_dir* is the directory where we start archiving from;
    i.e. *base_dir* will be the common prefix of all files and
-   directories in the archive.
+   directories in the archive.  *base_dir* must be given relative
+   to *root_dir*.
```

Comment:
```suggestion
   to *root_dir*.
   
   See :ref:`shutil-archiving-example-with-basedir` for how to use *base_dir* and *root_dir* together.
```

This is what I meant by "link to the example from the `make_archive()` documentation" in my earlier comment. Sorry if I wasn't clear :)